mounts/org.osbuild.ostree.deployment

```diff
@@ -98,32 +98,37 @@ class OSTreeDeploymentMount(mounts.MountService):
         ref = deployment["ref"]
         serial = deployment.get("serial", 0)
 
-        # create a private mountpoint for the tree, which is needed
-        # in order to be able to move the `root` mountpoint, which
-        # is contained inside tree, since "moving a mount residing
-        # under a shared mount is invalid and unsupported."
+        # The target path where we want the deployment to be mounted
+        # is the root of the tree.
+        target = tree
```

Comment:
Just to be clear, right now this commit has no functional effect, right?

Reply:
Ignore this.  I wrote it while I was reading the commits before moving on to the next one and forgot to delete it.